src/tutorial/furthertopics/suicide-triggers.rst

```diff
@@ -129,9 +130,11 @@ Open the ``cylc gui`` and run the suite::
    cylc gui suicide-triggers &
    cylc run suicide-triggers
 
-The suite will run for three cycles then get stuck. You should see something
-similar to the diagram below. As the ``bake_cake`` task fails randomly what
-you see might differ slightly. You may receive a "suite stalled" email.
+The suite will run for three cycles then get stuck (because of the
+:cylc:conf:`[scheduling]runahead limit`).
```

Comment:
The suite won't stall post-SoD, see https://github.com/cylc/cylc-flow/issues/3792. I think this whole page will need rewriting